client/my-sites/people/team-list/index.jsx

```diff
@@ -95,6 +112,7 @@ var Team = React.createClass( {
 		return (
 			<div>
 				<PeopleListSectionHeader
+					showRoles={ showRoles }
```

Comment:
Do you think this extra `prop` use useful outside of the case of disabling the new feature?